public/index.html

```diff
@@ -1,11 +1,70 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <title>07-photo-site</title>
-  <link rel="stylesheet" href="/app.css">
-</head>
-<body>
+  <head>
+    <meta charset="utf-8">
+    <title>07 PHOTO SITE</title>
+    <link rel="stylesheet" href="/app.css">
+  </head>
+  <body>
 
+    <div class="container">
+      <nav class="top-nav">
+        <p class="top-nav__group">
+          <i class="fa fa-arrow-left top-nav__icon"></i>
+          <span class="top-nav__slash">/</span>
+          <i class="fa fa-tint top-nav__icon"></i>
+
+          <strong class="top-nav__brand">Christian Zena</strong>
+          Photography
+        </p>
+        <p class="top-nav__group">
+          <a href="#" class="top-nav__link">About</a>
+          <a href="#" class="top-nav__link">Work</a>
+          <a href="#" class="top-nav__link">Contact</a>
+        </p>
+      </nav>
+    </div>
+
+    <header class="banner">
+      <img class="banner__image" src="http://www.84colors.com/blog/wp-content/uploads/2010/11/IMG_4088.jpg">
+      <h2 class="banner__text">Landscape</h2>
+    </header>
+
+    <div class="container">
+      <div class="row">
+        <div class="row__photo">
+          <div class="row__photo-info">
+          <img src="http://cdn.picturecorrect.com/wp-content/uploads/2009/10/beach-photography.jpg">
+          <h3 class="photo-info">Austin flannel slavia<span class="photo-date">
+              05/05/2015</span>
+          </h3>
+          <div class="row__photo-info-list">
+            <ul style="list-style-type:none">
```

Comment:
Don't have a `:` in class names.

Also you are doubling up here. The `ul` is already a separate group. Instead group around the image and h3.